Jenkinsfile

```diff
@@ -607,6 +607,48 @@ pipeline {
 
     }
 
+    stage('L2: Speech to Text AED') {
+      when {
+        anyOf {
+          branch 'r1.23.0'
+          changeRequest target: 'r1.23.0'
+        }
+      }
+      steps {
+        sh 'python examples/asr/speech_multitask/speech_to_text_aed.py \
+        model.prompt_format=canary \
+        model.model_defaults.asr_enc_hidden=256 \
+        model.model_defaults.lm_dec_hidden=256 \
+        model.encoder.n_layers=12 \
+        model.transf_encoder.num_layers=0 \
+        model.transf_decoder.config_dict.num_layers=12 \
+        model.train_ds.manifest_filepath=/home/TestData/asr/manifests/canary/an4_canary_train.json \
+        ++model.train_ds.is_tarred=false \
+        model.train_ds.batch_duration=60 \
+        +model.train_ds.text_field="answer" \
```

Comment:
In the manifest, why isnt `text` the key for the transcript - why is it now "answer"?

Reply:
AFAIK that was an already established convention in speech lm expts, in order to support these cases we allow users to choose whichever keys they want to read text and lang from.

Reply:
Ok  - but why is the default not `text` as has been nemos convention for the past 4 years? If this is arbitrary, please make it revert to the nemo default of reading `text` as the text field, whatever it may be. If user wants to change it, that's their prerogative, but nemos defaults are set as done before.